WordPress/Classes/ViewRelated/Comments/FullScreenCommentReplyViewController.swift

```diff
@@ -26,6 +26,7 @@ public class FullScreenCommentReplyViewController: EditCommentViewController, Su
     /// Reply Suggestions
     ///
     private var siteID: NSNumber?
+    private var prominentSuggestionsIds: [NSNumber]?
```

Comment:
I think there is a subtle semantic differences between the two.

`nil` array means the prominent suggestions feature is not used in some screens ( e.g GutenbergSuggestionsViewController ). I think it also saves some memory compared to empty array ( though the difference is probably negligible )

Whereas an empty array could mean that the view controller tried to find prominent suggestions but didn't find any.

That being said, the drawback of optional array is that we have to unwrap it every time we want to use the it. And this could be significant if we are using the property very frequently.

Happy to change it to empty array if that's your preference 🙏

Reply:
> nil array means the prominent suggestions feature is not used in some screens

I'm fine with this logic. 👍